tests/education/tochniyot/test_scraper.py

```diff
@@ -0,0 +1,24 @@
+from datapackage_pipelines_budgetkey.pipelines.education.tochniyot.scraper import TAARICH_STATUS, TaarichStatus_to_date, \
+    send_tochniyot_request
+
+
+def test_TaarichStatus_to_date():
+    """
+    test the conversion from JS date format to readable date format
+    :return:
+    """
+    row = {TAARICH_STATUS: '/Date(23424234234234)/'}
+    TaarichStatus_to_date(row)
+    assert row[TAARICH_STATUS] == '14/04/2712 19:43:54'
+
+    row = {TAARICH_STATUS: '/Date(-62135596800000)/'}
```

Comment:
Try to avoid dates before 1/1/1000 as I saw that there are major differences between platforms regarding the way they're formatted (and for the tests it doesn't really matter).
I think that's also the reason that the tests are failing.

Reply:
Done